README.md

```diff
@@ -1,3 +1,7 @@
+This is a fork of [tmuxinator/tmuxinator](https://github.com/tmuxinator/tmuxinator). In this fork I am trying to add some new features. If a feature is ready, I will send it back as a pull request.
```

Comment:
@geggo98 Was this added by mistake? I think this should not be part of a PR. :smile: